app/tests/spec/models/account.js

```diff
@@ -509,6 +572,30 @@ function (chai, sinon, p, Constants, Assertion, ProfileClient,
       });
     });
 
+    describe('with an unverified account', function () {
+      beforeEach(function () {
+        sinon.stub(account, 'profileClient', function () {
+          return p.reject(AuthErrors.toError('UNVERIFIED_ACCOUNT'));
+        });
+      });
+
+      ['getAvatar', 'getAvatars', 'postAvatar', 'deleteAvatar', 'uploadAvatar']
```

Comment:
This list is duplicated in [multiple places](https://github.com/mozilla/fxa-content-server/pull/2933/files#diff-0a7ecb99efaded3ca217a9ef51bcf4b4R557), should it be made into a constant in case there are functions added/removed in the future?